zone/client.cpp

```diff
@@ -7483,21 +7483,51 @@ void Client::RemoveAutoXTargets()
 
 void Client::ShowXTargets(Client *c)
 {
-	if(!c)
+	if (!c) {
 		return;
+	}
+
+	auto xtarget_count = 0;
+
+	for (int i = 0; i < GetMaxXTargets(); ++i) {
+		c->Message(
+			Chat::White,
+			fmt::format(
+				"XTarget Slot {} | Type: {} ID: {} Name: {}",
```

Comment:
It'd be nice to keep consistent debugging outputs

Eg

```
xtarget slot [{}]
```

vs

```
XTarget Slot {} |

Reply:
All my other stuff is the same format as this. This isn’t a debug log, it’s a client message.

Reply:
It's a client message used in the same sense, it's printing variables used for debugging